ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -980,6 +980,43 @@ def test_tensorflow_gather_nd(
     )
 
 
+# tile
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    dtypes_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full="True")),
+    dt_and_multiples=helpers.dtype_and_values(
+        available_dtypes=st.sampled_from(["int32", "int64"])
```

Comment:
The `TypeError: i is not a valid data type that can be generated, only integers, floats and booleans are allowed.` is because `st.sampled_from` returns a string (either "int32" or "int64"), but `available_dtypes` should be a list.
I think that simply setting `available_dtypes=["int32", "int64"]` would work fine in this case.